invenio_curations/notifications/builders.py

```diff
@@ -21,79 +31,99 @@
 class CurationRequestActionNotificationBuilder(NotificationBuilder):
     """Notification builder for curation actions."""
 
-    type = "curation-request"
-
-    @classmethod
-    def build(cls, identity, request):
-        """Build notification with request context."""
-        return Notification(
-            type=cls.type,
-            context={
-                "executing_user": EntityResolverRegistry.reference_identity(identity),
-                "request": EntityResolverRegistry.reference_entity(request),
-            },
-        )
+    type: ClassVar[str] = "curation-request"
 
-    context = [
+    context: ClassVar[list[EntityResolve]] = [
```

Comment:
should we use the base class here? https://github.com/inveniosoftware/invenio-notifications/blob/master/invenio_notifications/services/generators.py#L19

should we use everytime the base class in such contexts?

Reply:
i think you did that for the `RecipientGenerator`